chipsec/utilcmd/cmos_cmd.py

```diff
@@ -70,27 +70,27 @@ def requires_driver(self):
 
         return True
 
-    def cmos_dump(self):
+    def cmos_dump(self) -> None:
         self.logger.log("[CHIPSEC] Dumping CMOS memory..")
         self._cmos.dump()
 
-    def cmos_readl(self):
+    def cmos_readl(self) -> None:
         val = self._cmos.read_cmos_low(self.offset)
-        self.logger.log("[CHIPSEC] CMOS low byte 0x%X = 0x%X" % (self.offset, val))
+        self.logger.log(f'[CHIPSEC] CMOS low byte 0x{self.offset} = 0x{val}')
```

Comment:
If I understand the code correctly, this formats integers (`self.offset` and `val`) as decimal, with a hexadecimal prefix. This seems buggy, and the code would likely be instead:
```python
self.logger.log(f'[CHIPSEC] CMOS low byte 0x{self.offset:X} = 0x{val:X}')
```

What do you think?

Reply:
Agreed.